### PR TITLE
Add a recommendation for running etcd at scale

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds info in the Scalability and Performance guide
about the maintenance operations needed for etcd to perform better.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
@ahardin-rh @akrzos @ashishkamra @mffiedler PTAL.

File: scaling_performance/host_practices.adoc [hunk 1]
```diff
@@ -169,6 +169,8 @@ endif::[]
 etcd processes are typically memory intensive. Master / API server processes are CPU intensive. This makes them a reasonable co-location pair within a single machine or virtual machine (VM). Optimize communication between etcd and master hosts either by co-locating them on the same host, or providing a dedicated network.
 ====
 
+For a large and dense cluster, etcd may suffer from poor performance if the keyspace grows excessively large and exceeds the space quota. Periodic maintaince of etcd including defragmentaion needs to be done to free up the space for the backend. It is highly recommended to monitor prometheus for etcd metrics and defrag it when needed before etcd raises a cluster-wide alarm that puts the cluster into a maintenance mode which only accepts key reads and deletes. Some of the key metrics to monitor are `etcd_server_quota_backend_bytes` which is the current quota limit, `etcd_mvcc_db_total_size_in_use_in_bytes` indicates the actual database usage after a history compaction while `etcd_debugging_mvcc_db_total_size_in_bytes` shows the database size including free space waiting for defragmentation.
```
Comment (mffiedler) [inline scaling_performance/host_practices.adoc]: Suggestions:  
- change "free up space in the backend" to "free up space in the data store"
- capitalize Prometheus
- `etcd_mvcc_db_total_size_in_use_in_bytes` which indicates
- change "while `etcd_debugging_mvcc_db_total_size_in_bytes` shows" to "and `etcd_debugging_mvcc_db_total_size_in_bytes` which shows"

**Review by mffiedler — CHANGES_REQUESTED** [review]

File: scaling_performance/host_practices.adoc (hunk 1 above)
Comment (chaitanyaenr) [reply scaling_performance/host_practices.adoc]: Made the changes requested. PTAL, thanks.

**Comment by ahardin-rh** [comment]
@chaitanyaenr If this is for the 4.1 release, you will need to resubmit this PR against the enterprise-4.1 branch instead of master

**Comment by chaitanyaenr** [comment]
@ahardin-rh this is for the 4.1 release, will create a PR against the enterprise-4.1 branch. Thanks.

**Pull request closed without merging**